pastas/rfunc.py

```diff
@@ -936,27 +948,17 @@ def get_tmax(self, p, cutoff=None):
 
     @staticmethod
     def gain(p):
-        print('Gain for x = L/2 :')
-        return np.pi**2 * p[1] / (8 * p[0])
-
-    @staticmethod
-    @njit
-    def step_kleur(p, t):
-        s = np.zeros(len(t))
-        for i, t_value in enumerate(t):
-            s_part = np.array([0.0])
-            for n in np.arange(1, p[3], 2):
-                s_part = np.append(
-                    s_part, ((1 / n**3) *
-                             (p[1] - p[1] * np.exp(-n**2 * t_value / p[1]))
-                             * np.sin(n * np.pi * p[2]))
-                )
-            s[i] = 4 / (np.pi * p[0]) * np.sum(s_part)
-        return s
+        return p[0]
 
     def step(self, p, dt=1, cutoff=None, maxtmax=None):
         t = self.get_t(p, dt, cutoff, maxtmax)
-        return self.step_kleur(np.asarray(p), t)
+        h = 0
+        for n in range(10):
```

Comment:
As @mbakker7 suggested, can you make `n` a parameter in the function `step_kleur`? So the response function gets an attribute self.n=10 which could be changed for research purposes.